swarm/pss/pss.go

```diff
@@ -34,6 +34,10 @@ const (
 	defaultSymKeyLength         = 32
 )
 
+var (
+	addressLength = len(pot.Address{})
```

Comment:
What is the concrete benefit of bits here? I understand it would give more fine-grained  control of the po check in the iterator, but I really think that's overkill.